metrics/web/lib/common/presentation/metrics_theme/model/dark_metrics_theme_data.dart

```diff
@@ -285,6 +285,15 @@ class DarkMetricsThemeData extends MetricsThemeData {
                 fontWeight: FontWeight.w500,
               ),
             ),
+            inactiveLoginOptionButtonStyle: MetricsButtonStyle(
+              color: ColorConfig.inactiveColor,
+              hoverColor: ColorConfig.inactiveColor,
+              labelStyle: TextStyle(
```

Comment:
Seems like we should use `MetrixTextStyle` as there is a line-height in designs.